server/jetstream.go

```diff
@@ -219,6 +219,7 @@ func (s *Server) EnableJetStream(config *JetStreamConfig) error {
 	cfg := *config
 	if cfg.StoreDir == _EMPTY_ {
 		cfg.StoreDir = filepath.Join(os.TempDir(), JetStreamStoreDir)
+		s.Warnf("Temporary storage directory used, data will be gone after OS restart")
```

Comment:
"Temporary storage directory used, data could be lost on system reboot"

Reply:
Done